src/drivers/bitbucket_cloud.js

```diff
@@ -170,6 +197,22 @@ class BitBucketCloud {
     return await response.json();
   }
 
+  async paginatedRequest(opts = {}) {
+    const { method = 'GET', body } = opts;
+    const result = await this.request(opts);
+
+    if (result.next) {
+      const next = await this.paginatedRequest({
+        fullEndpoint: result.next,
```

Comment:
assign previously

Reply:
why not?

Reply:
Because I think that checking for `next` conveys better the purpose of the algorithm than checking for `url` in the conditional.